src/exceptiongroup/_formatting.py

```diff
@@ -51,6 +51,7 @@ def traceback_exception_init(
         _seen=_seen,
         **kwargs,
     )
+    self.__note__ = getattr(exc_value, "__note__", None) if exc_value else None
```

Comment:
Do we want to set `__note__` even if it doesn't exist in the source exception?

Reply:
Yeah, treating a nonexistent note as None simplifies the later code considerably, and is semantically consistent with both new and old behaviour.